src/EducationService.Data/UserEducationRepository.cs

```diff
@@ -6,6 +6,8 @@
 using Microsoft.AspNetCore.JsonPatch;
 using Microsoft.EntityFrameworkCore;
 using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
do we need this? these are the only changes to this file - didn't it work before without them?

Reply:
just useless link